gamestonk_terminal/alternative/covid/covid_controller.py

```diff
@@ -0,0 +1,415 @@
+    def call_help(self, _):
+        """Process help command"""
+        self.print_help()
+
+    def call_quit(self, _):
+        """Process quit menu command"""
+        t_console.print("")
+        self.queue.insert(0, "quit")
+
+    def call_exit(self, _):
+        """Process exit terminal command"""
+        self.queue.insert(0, "quit")
+        self.queue.insert(0, "quit")
+        self.queue.insert(0, "quit")
+
+    def call_reset(self, _):
+        """Process reset command"""
+        self.queue.insert(0, "covid")
+        self.queue.insert(0, "alternative")
+        self.queue.insert(0, "reset")
+        self.queue.insert(0, "quit")
+        self.queue.insert(0, "quit")
+
+    def call_country(self, other_args: List[str]):
+        """Process country command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="country",
+            description="Select a country to look at data for.",
+        )
+        parser.add_argument(
+            "-c",
+            "--country",
+            nargs="+",
+            type=str,
+            dest="country",
+            help="Country to get data for.",
+        )
+        if other_args and "-" not in other_args[0][0]:
+            other_args.insert(0, "-c")
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if ns_parser:
+            country = " ".join(ns_parser.country)
+            if country not in self.COUNTRY_LIST:
+                t_console.print(f"{country} not a valid selection.\n")
+                return
+            self.country = country
+        t_console.print("")
+
+    @try_except
+    def call_ov(self, other_args: List[str]):
+        """Process hist command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="ov",
+            description="Show historical cases and deaths by country.",
+        )
+        ns_parser = parse_known_args_and_warn(
+            parser,
+            other_args,
+            export_allowed=EXPORT_BOTH_RAW_DATA_AND_FIGURES,
+            raw=True,
+            limit=10,
+        )
+        if ns_parser:
+            covid_view.display_covid_ov(
+                self.country,
+                raw=ns_parser.raw,
+                limit=ns_parser.limit,
+                export=ns_parser.export,
+            )
+
+    @try_except
+    def call_rates(self, other_args: List[str]):
+        """Process hist command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="rates",
+            description="Show historical rates country.",
+        )
+        ns_parser = parse_known_args_and_warn(
+            parser,
+            other_args,
+            export_allowed=EXPORT_BOTH_RAW_DATA_AND_FIGURES,
+            raw=True,
```

Comment:
I think this was a significant change to be in a different PR. Because it's not related at all with alternative data. Plus, in that PR we could go back and tackle all functions that needed such.

Reply:
This isn't really a significant change.  Since I added default arguments to the function, it won't break anything and we can go back and revisit where we see as needed.